### PR TITLE
handle permission requests for multiple permissions

### PR DESCRIPTION
**Pull request by bridiver**
prereq for https://github.com/brave/browser-laptop/issues/10864

### CODE REVIEW

File: brave/browser/brave_permission_manager.cc [hunk 1]
```diff
@@ -63,19 +66,30 @@ int BravePermissionManager::RequestPermission(
...
+    const GURL& requesting_origin,
+    bool user_gesture,
+    const base::Callback<void(
+    const std::vector<blink::mojom::PermissionStatus>&)>& response_callback) {
   int render_frame_id = MSG_ROUTING_NONE;
   int render_process_id = MSG_ROUTING_NONE;
   GURL url;
   // web notifications do not currently have an available render_frame_host
   if (render_frame_host) {
+    LOG(ERROR) << "got render frame host";
```
Comment (bbondy) [inline brave/browser/brave_permission_manager.cc]: nit: left over logging

File: brave/browser/brave_permission_manager.cc [hunk 2]
```diff
@@ -63,19 +66,30 @@ int BravePermissionManager::RequestPermission(
     const GURL& requesting_origin,
     bool user_gesture,
     const base::Callback<void(
-        blink::mojom::PermissionStatus)>& response_callback) {
+          blink::mojom::PermissionStatus)>& response_callback) {
+  LOG(ERROR) << "request permission " << (permission == content::PermissionType::GEOLOCATION);
```
Comment (posix4e) [inline brave/browser/brave_permission_manager.cc]: is this also an accidental log

**Comment by posix4e** [comment]
How do I know this works?

**Comment by darkdh** [comment]
++

**Pull request merged**